frontend_lib/src/css/Generic.styl

```diff
@@ -12,3 +12,7 @@
   background none
   &:active
     outline none
+
+.dropdownMenuSeparatorLine
+  padding: 0
+  margin: 0
```

Comment:
you can omit the ":"
it isn't mandatory when using stylus and coding rule of tracim is to not write them
it makes the code cleaner

Reply:
done